manifests/h/h3poteto/whalebird-desktop/4.7.3/h3poteto.whalebird-desktop.installer.yaml

```diff
@@ -0,0 +1,42 @@
+# Created with YamlCreate.ps1 v2.2.1 using InputObject 🤖 $debug=QUSU.CRLF.7-2-7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: h3poteto.whalebird-desktop
+PackageVersion: 4.7.3
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2022-12-18
+Installers:
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://github.com/h3poteto/whalebird-desktop/releases/download/4.7.3/Whalebird-4.7.3-windows-x64.exe
+  InstallerSha256: 4BD577109427A2340B899EA2190D553FC36F70F7D90C3CA959CF838B51C29F21
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x86
+  Scope: machine
+  InstallerUrl: https://github.com/h3poteto/whalebird-desktop/releases/download/4.7.3/Whalebird-4.7.3-windows-x64.exe
+  InstallerSha256: 4BD577109427A2340B899EA2190D553FC36F70F7D90C3CA959CF838B51C29F21
+  InstallerSwitches:
+    Custom: /ALLUSERS
+- Architecture: x64
```

Comment:
```suggestion
- Architecture: ia32
```

Reply:
I'm not sure if Winget supports ia32, or if that architecture has a different internal name. This section, and the following section, might need to be removed to pass validation.

Reply:
> I'm not sure if Winget supports ia32, or if that architecture has a different internal name. This section, and the following section, might need to be removed to pass validation.

ia32 isn't part of the enum for supported architectures; x86 is the correct architecture to use for 32 bit support